packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -607,6 +631,12 @@ const MultiSelect = (props: Props) => {
     const filteredItems = getMenuItems(allChildren);
     const isDisabled = numEnabledOptions === 0 || disabled;
 
+    React.useEffect(() => {
+        if (open) {
```

Comment:
The equivalent logic in SingleSelect doesn't check for `open`, do we want to include it there too? If I'm understanding correctly, this would make sure the message isn't announced when the length of filteredItems changes while the MultiSelect is closed!

Reply:
Done!